### PR TITLE
Dry up Searchable Snapshots ITs

### PR DESCRIPTION
**Pull request by original-brownbear**
Just a few spots where we can dry up these tests using the snapshot test infrastructure
in core that I found while studying the existing searchable snapshot tests. Also, using the helper methods from the core infrastructure helps add quite a bit of useful debug logging that is always nice to have when investigating failures.

### CODE REVIEW

File: x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/blobstore/cache/SearchableSnapshotsBlobStoreCacheIntegTests.java [hunk 1]
```diff
@@ -115,9 +115,9 @@ public void testBlobStoreCache() throws Exception {
 
         final String repositoryName = randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
         final Path repositoryLocation = randomRepoPath();
-        createFsRepository(repositoryName, repositoryLocation);
+        createRepository(repositoryName, "fs", repositoryLocation);
```
Comment (tlrx) [inline x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/blobstore/cache/SearchableSnapshotsBlobStoreCacheIntegTests.java:118]: We have a static constant F`sRepository.TYPE` for this why nobody wants to use it? :)

**Review by tlrx — APPROVED** [review]
LGTM

File: x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/blobstore/cache/SearchableSnapshotsBlobStoreCacheIntegTests.java (hunk 1 above)
Comment (original-brownbear) [reply x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/blobstore/cache/SearchableSnapshotsBlobStoreCacheIntegTests.java:118]: I don't know :) but it's a good point ... I'll keep it in mind next time I touch this stuff, promise :)

**Comment by original-brownbear** [comment]
Thanks Tanguy!

**Pull request merged**